src/index.html

```diff
@@ -17,6 +29,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo_link"
+      >
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        />
```

Comment:
The `alt` attribute should describe the image content. Instead of 'logo', provide a more descriptive alternative text that describes the logo, such as the company name or the brand the logo represents.